.github/CODEOWNERS

```
@@ -8,3 +8,5 @@ codetransform/ @aws/elastic-gumby
 codewhisperer/ @aws/codewhisperer-team
 
 mynah-ui/ @aws/dexp
+mynah-ui/package.json @aws/dexp @aws/elastic-gumby @aws/earlybird
```

would be nice if we could do:
```suggestion
mynah-ui/{package,package-lock}.json @aws/dexp @aws/elastic-gumby @aws/earlybird
```

but doesn't seem like that's supported